project-management/stand-ups.md

```diff
@@ -0,0 +1,39 @@
+# Squad stand-ups
+
+Each squad must have a [stand-up meeting](https://en.wikipedia.org/wiki/Stand-up_meeting) every morning, with the possible exception of end-of-iteration demo days.
+
+## Guidelines
+
+All stand-ups should try to follow these guidelines, as well as [general best practice](https://martinfowler.com/articles/itsNotJustStandingUp.html).
+
+- Stand-ups should be as quick a possible and no longer than 15min
+- Stand-ups should happen each day at the same time each day, even if a team member is late
+
+### Standard format
+
+During a stand-up, each team member should mention:
```

Comment:
We agreed on the off-site that all squads will now try going through column-by-column, rather than person-by-person. I think this should probably be reflected here. I'll have a go at rewording it in a bit.

Reply:
@nottrobin I already updated the issue but feel free to re-word it

Reply:
@kasiag I should teach you at some point how to do your own pull request